base/doc/ltnews41.tex

```diff
@@ -772,283 +850,305 @@ \subsection{A version of \cs{input} for expansion contexts}
+changed later on. 
+%
+This has been corrected: these arguments now get fully
+expanded when the declaration 
+is made.
+%
+\githubissue{1675}
+
+
+
+\subsection{Correction in the float placement algorithm}
 
 When floats are added to the current or next page, \LaTeX{} makes
-several tests to find an area that can receive the float. One of these
+several tests in order to find an area that can receive the float. One of these
 tests calculates how much space is already used on the page and how
 much additional space is needed to place the float in a particular
 area. This means that it looks not only at the height of the float but
 also at the values from \cs{intextsep} (for \texttt{h} floats) or
 \cs{textfloatsep} and \cs{floatsep} (for \texttt{t} and \texttt{b}
 floats). The resulting space requirement is then stored in an internal
 variable and compared to the space still available on the page.
+If the test fails, the algorithm tries the next area. 
 
-If the test fails, the algorithm tries the next area. Unfortunately,
-it was reusing the value in that internal variable as the starting
-point for the next test without removing the added space for the float
+Unfortunately, the code 
+was reusing the value in that internal variable as the starting
+point for the next test, without removing the added space for the float
 separation (\cs{intextsep}, \cs{floatsep}, or \cs{textfloatsep}). Thus
 the comparison was being made with the wrong value (i.e., too high);
 therefore the test may have incorrectly concluded that a float doesn't
-fit, even though in fact it did.
-
+fit, even when, in fact, it did fit. 
 This has now been corrected.
 %
 \githubissue{1645}
 
 
 \subsection{Correct \cs{CheckEncodingSubset}}
 
-In \cite{41:ltnews36} and again in \cite{41:ltnews39} we suggested
+In \cite[p.\,83]{41:ltnews}, and again in \cite[p.\,100]{41:ltnews}, we suggested
 that font maintainers should place an appropriate
-\cs{DeclareEncodingSubset} declaration into each
-\texttt{ts1\meta{family}.fd} file so that it is tied to the font
-definition and available if a font family is explicitly selected
-through \cs{fontfamily}\texttt{\{\meta{name}\}} instead of using some
-font support package. Unfortunately, doing this could result in
-incorrectly selected glyphs when the font encoding subset setting was
-evaluated before the \texttt{.fd} file was loaded (because then subset
-9 was assumed).  This has now been corrected and
-\cs{CheckEncodingSubset} now first loads the \texttt{.fd} file, if
+\cs{DeclareEncodingSubset} declaration in each
+\texttt{ts1\meta{family}.fd} file, so that this is tied to the font
+definition and so will be available whenever a font family is explicitly selected
+by \cs{fontfamily}\texttt{\{\meta{name}\}}
+%%FMi sorry but this is not correct: we should not claim that the work of font package maintainers is useless! It is not correct
+%. This is better than using a
+instead of using a
+font support package.
```

Comment:
But you need to explain more clearly what is used instead of what! 

Or, Maybe: 
with no need to use a . . .